lib/http-bridge/test/integration/Test/Integration/Scenario/Transactions.hs

```diff
@@ -222,10 +225,9 @@ spec = do
                 ]
 
     it "TRANS_CREATE_02 - Multiple Output Txs don't work on single UTxO" $ \ctx -> do
-        wSrc <- fixtureWalletWith ctx [200_000]
+        wSrc <- fixtureWalletWith ctx [20_000_000]
```

Comment:
What's the rationale here :thinking: ?

Reply:
I thought it might be too less for potential fee for multi-transaction. (But perhaps it isn't). So the rationale was to make sure we have enough funds to cover but not enough UTxO. Anyway changing to a value _closer_ to real live (still potentially enough to cover transaction)... `2_124_333`

Reply:
That's what I thought :+1:. THanks for clarifying.